Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+    React is a JavaScript library for building User Interfaces. It tries to solve the problem of handling state in large, complex applications. It does this by breaking down its apps into self-contained components that are also reusable.
 
-1.  What does it mean to _think_ in react?
+2.  What does it mean to _think_ in react?
+    Thinking in react means to think of UI in components and which of those components are functional and which of those components hold state.
 
-1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+3.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+    One of the biggest differences is that a Class component is a JS class and a Stateful component is a function.
 
-1.  Describe state.
+4.  Describe state.
+    State is data passed down through an application.

```

Comment:
I would say state does not necessarily need to be passed down. It is just the dynamic aspect of a component, as represented in a data object over the component's life cycle... or something like that.